src/main.rs

```diff
@@ -79,6 +79,11 @@ fn main() {
 	let cache = Arc::new(Mutex::new(cache.unwrap_or_default()));
 	let config = Rc::new(RefCell::new(config.unwrap_or_default()));
 
+	if args.absolute {
+		config.borrow_mut().title.get_or_insert_with(|| Default::default()).displayed_folders =
+			Some(100_000);
+	}
+
```

Comment:
Instead of or along with the absolute option please make a command line option that allows specifying the visible folders from the path, as that was the request in #8 and I think it could be useful. If you keep the absolute option the assigned number here could be `std::u32::MAX`. I know that no-one will have a path consisting of more than a hundred thousand components but I think max looks nice in such places.

Reply:
The latest PR adds a `--folders` option, abbreviated as `-f`, so you can add `-f3` to show 3 folders, for example.